### DISCUSSION TITLE
What is wrong with this user pool?

### DISCUSSION THREAD

**Discussion by shieldsrholding** [RainbowMiner/RainbowMiner#2917] [Q&A]
I m using AikaPool as an example. I just want to point my rig at a pool and mine RandomX, where the coin is determined by the pool port.  For testing purposes, I am using SISPOP, port 5905 low diff.

{
    "Enable": "1",
    "Name": "AikaPool",
    "PoolFee": "0",
    "Algorithm": "RandomX",
    "CoinName": "",
    "CoinSymbol": "",
    "Currency": "",
    "Protocol": "stratum+tcp",
    "Host": "stratum.aikapool.com",
    "Port": "5905",
    "User": "$Wallet.$WorkerName",
    "Pass": "x",
    "Region": "US",
    "SSL": "0",
    "EthMode": ""
}

**Comment by RainbowMiner** [comment]
Add a CoinSymbol (=coin that is mined) and a Currency (=coin that will be paid out). For your pool and port that would be:
```
  "CoinSymbol": "SISPOP",
  "Currency": "SISPOP",
```

**Comment by shieldsrholding** [comment]
So, this does not work for me. I updated the userpool to reflect the coin as mentioned above. If I direct the unit to mine only this pool it says, "WARNING: No pools available: check your configuration".
